data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -219,6 +219,33 @@
     "Michigan EJSCREEN Priority Community"
 )
 
+# Maryland EJSCREEN Data.
+MARYLAND_EJSCREEN_TRACT_25_PERCENT_FIELD: str = (
```

Comment:
Nit: a percent is a very different thing than a percentile, so can we call all of these `_PERCENTILE_FIELD` instead of `_PERCENT_FIELD`?

Reply:
Good point - I will make the change here.